org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/typesystem/arguments/IFeatureCallArgumentSlot.java

```diff
@@ -17,27 +17,26 @@
  * @noextend This interface is not intended to be extended by clients.
  * @noimplement This interface is not intended to be implemented by clients.
  */
-@SuppressWarnings("all")
 public interface IFeatureCallArgumentSlot {
-  boolean isVarArg();
-  
-  boolean isSuperfluous();
-  
-  /**
-   * The list of expressions. The list is never <code>null</code> but may contain
-   * <code>null</code> entries.
-   */
-  List<XExpression> getArgumentExpressions();
-  
-  /**
-   * @Nullable
-   */
-  XExpression getArgumentExpression();
-  
-  /**
-   * @Nullable
-   */
-  LightweightTypeReference getDeclaredType();
-  
-  void markProcessed();
+	boolean isVarArg();
+
+	boolean isSuperfluous();
+
+	/**
+	 * The list of expressions. The list is never <code>null</code> but may
+	 * contain <code>null</code> entries.
+	 */
+	List<XExpression> getArgumentExpressions();
+
+	/**
+	 * @Nullable
```

Comment:
Don't we have the actual `@Nullable` available?

Reply:
unfortunately not. at least not ubiquitous

Reply:
Sad life